cmd/airflow.go

```diff
@@ -491,12 +492,18 @@ func newObjectExportCmd() *cobra.Command {
 
 // Use project name for image name
 func airflowInit(cmd *cobra.Command, args []string) error {
+	pwd, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+
+	if len(args) > 0 {
+		projectName = args[0]
+		config.WorkingPath = filepath.Join(config.WorkingPath, projectName)
+	}
+
 	// Validate project name
 	if projectName != "" {
-		// error if project name has spaces
-		if len(args) > 0 {
-			return errProjectNameSpaces
-		}
```

Comment:
This isn't needed because we already use cobra's NumberOfArguments configuration on this command. Anything over 1 argument shows the help message so this condition is never reached.